deploy-as-code/helm/environments/uat.yaml

```diff
@@ -452,6 +452,13 @@ dashboard:
       sub_filter  '<head>' '<head>
       <script src=https://s3.ap-south-1.amazonaws.com/egov-dev-assets/globalConfigs.js type=text/javascript></script>
       ';"
+
+sanitation-ui:
+  custom-js-injection: |
+    sub_filter.conf: "
+      sub_filter  '<head>' '<head>
+      <script src=https://s3.ap-south-1.amazonaws.com/egov-dev-assets/globalConfigsFSM.js type=text/javascript></script>
```

Comment:
create a new global config for uat , since UAT has state tenant as PG

Reply:
Updated @jagankumar-egov